ford_fusion_sehybrid_2019/drivers.launch

```diff
@@ -43,39 +43,17 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="mock_lightbar"       value="$(eval 'lightbar' in arg('mock_drivers').split())"/>
 
   <!-- Specific Drivers -->
-  <arg name="drivers" default="ssc_interface_wrapper lightbar_driver avt_vimba_camera"
+  <arg name="drivers" default="ssc_interface_wrapper lightbar_driver"
```

Comment:
Same comment as above regarding lightbar driver and unrequired mock_drivers

Reply:
Done